config/config.php

```diff
@@ -166,7 +168,7 @@
     'cache' => [
         'enabled' => false,
         'key' => 'laravel-modules',
-        'lifetime' => 60,
+        'lifetime' => 60
```

Comment:
Why? Trailing commas is pretty much a PSR requirement, which this package follows.